doc/paq-nvim.txt

```diff
@@ -477,15 +477,19 @@ If you've checked all that and still think that the package _has_ to be lazy
 loaded, you can declare the package as optional, and call |packadd| when
 needed.
 
-You can use an |autocommand| to do the lazy loading, but currently there isn't
-a way to declare an autocommand using only Lua.
+You can use an |autocommand| to do the lazy loading:
```

Comment:
Suggestion: This could mention the version in which the lua autocommand feature was added:

> Since Neovim version 0.7, you can write an |autocommand| in Lua to do the lazy loading: